src/Interfaces/VTK.jl

```diff
@@ -7,9 +7,31 @@ using Reexport
 using WriteVTK
 
 using ..Meshes
+using ..Nodes
+using ..StructuralAnalyses
 
 @reexport import WriteVTK: vtk_grid
+export write_vtks
 
-# Add code here.
+function write_vtks(states_sol, filename)
```

Comment:
while at this, maybe add a docstring mentioning the fields currently exported? (so displacements for now)

Reply:
good idea!

Reply:
ie.

```julia
"""
Generate a VTK file given a solution struct.
Currently only `displacements` are exported for each time point.
"""
function write_vtk(...) # see comment above
```